articles/internet-protocols/index.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/internet-protocols/
+title: Understanding IPV4 and IPV6 protocols
+description: This article will be a walk through the IPv4 and IPv6 Internet protocols which make communication on the internet possible. 
+author: mackrine-awino
+date: 2021-05-05T00:00:00-13:00
+topics: [Protocols]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/internet-protocols/hero.jpg
+    alt: Internet Protocols Hero Image
+---
+Internet protocols are what make communication on the internet possible. Every device connected to the internet is assigned an IP address. This enables the identification and location of networked computers on the internet. 
+<!--more-->
+Internet protocols also enable the transportation of data items on the internet. Datagrams are transmitted from one device to another without connections to one another by using internet protocols. The datagrams are transmitted with utmost independence from all others.
```

Comment:
Briefly introducing what are datagrams before explaining how they are transmitted from one device to another